net.sourceforge.xournal.json

```diff
@@ -0,0 +1,80 @@
+{
+  "app-id": "net.sourceforge.xournal",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "sdk": "org.gnome.Sdk",
+  "command": "xournal",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--filesystem=host"
+  ],
+  "modules": [
+    {
+      "name": "libart",
+      "cleanup": [
+        "/bin"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libart_lgpl/2.3/libart_lgpl-2.3.21.tar.gz",
+          "sha256": "2a10b1c884bb37c7bd5c5eba59dbd0d28bf68e64a42d7f2083c836f60f8e98fa"
+        }
+      ]
+    },
+    {
+      "name": "libgnomecanvas",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libgnomecanvas/2.30/libgnomecanvas-2.30.3.tar.gz",
+          "sha256": "a8ca85e734ab03ecf1fba7d99e01ae2541d0df539c69a7da9414cde928c116da"
+        }
+      ]
+    },
+    {
+      "name": "poppler",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DCMAKE_INSTALL_LIBDIR=/app/lib",
```

Comment:
The LIBDIR is not needed anymore

Reply:
Copied that from another file, will remove it.

Reply:
I think there are conflicting reports about this so do verify the results.

Reply:
@TingPing It successfully builds with removed LIBDIR.